### ISSUE TITLE
training loss=nan

### ISSUE THREAD

**Issue by Laudarisd** [abdelrahman-gaber/tf2-object-detection-api-tutorial#8]
Thanks for the nice tutorial. I was able to train my custom datasets but got loss=Nan. 
I also decreased the learning rate but couldn't fix the problem?

Any suggestions?
